python/spec/fixtures/pipfile_files/version_not_specified.lock

```diff
@@ -1,12 +1,13 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "c402ea48092e9d467af51a483bb8dd8ad0620e11c94f009dcd433f97a99d45db"
+            "sha256": "e76ae491d793d659f05c7f7eab261fd6167dc062efcba08f17be68e73eb87665"
```

Comment:
This was the only lockfile that the tests required to have an updated hash.

I strongly suspect the hashes are outdated for the other fixture lockfiles, but again, we can regenerate them all after updating to latest `pipenv` once we drop Python 3.7.